### PR TITLE
3/fix widget control focus

### PR DESCRIPTION
**Pull request by abea**
Resolves https://apostrophecms.atlassian.net/browse/A30U-683, "Focus is being stolen by other widgets"

In a two column widget, you should now have no trouble accessing widget controls for Widget A with your mouse that happen to appear on top of Widget B. All other widget control operations should work as they have.

### CODE REVIEW

**Comment by abea** [comment]
I recommend reviewing the first commit on its own. Those are the meaningful changes. The CSS in this file had repetition and inconsistent organization, so I cleaned up a bit of that in the second commit.

File: modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue [hunk 1]
```diff
@@ -428,61 +428,69 @@ export default {
...
-      z-index: $z-index-widget-highlight;
       &:before, &:after {
         opacity: 0.4;
       }
     }
     &.apos-focus {
-      z-index: $z-index-widget-focus;
       &:before, &:after {
         opacity: 1;
         border-top: 1px solid var(--a-primary);
```
Comment (boutell) [inline modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue:446]: Not an issue to hold this PR for, but have we bogarted the entire "a" namespace for CSS variables? Or is it just a SASS thing?

**Review by boutell — APPROVED** [review]
It does feel better.

File: modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue (hunk 1 above)
Comment (abea) [reply modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue:446]: we're bogarting it.

**Pull request merged**